app/models/ability.rb

```diff
@@ -54,13 +54,15 @@ def initialize(user, token = nil)
     owned_orgs = Organization.with_role(:owner, user).pluck(:id)
     can :manage, Organization, id: owned_orgs
     can %i[crud profile info], [Stream, Upload], organization: { id: owned_orgs }
+    cannot %i[reanalyze], [Stream], organization: { id: owned_orgs }
```

Comment:
I don't think this in needed. Org owners cannot re-analyze streams even if this line is not present because they can `create` and `read` streams, but don't have full `manage` permissions.

Reply:
@corylown I removed this line.